spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -758,7 +758,7 @@
         context "when qualifies for income exemption" do
           it "does not check 53c Schedule NJ-HCC checkbox and leaves 53a, 53b, and 53c amount blank" do
             single_income_threshold = 10_000
-            allow_any_instance_of(Efile::Nj::Nj1040Calculator).to receive(:calculate_line_54).and_return single_income_threshold
+            allow_any_instance_of(Efile::Nj::Nj1040Calculator).to receive(:calculate_line_29).and_return single_income_threshold
```

Comment:
[dust] can you help me understand why this change was needed? seems very separate from Line 15/29

Reply:
upgrading to [pebble/sand] - Out of interest I tried running this file of tests the way had it been before, with line 54, and it still passes that way... as well as with lines 760 and 761 removed entirely!

So this test doesn't seem to be testing what we think it's testing, which still might be a separate bug from this ticket but we may want to resolve while we are looking at it now, but at least make a follow-up bug ticket.

My guess is we need to add a `context "when does not qualify for income exemption" do` next to this test, making sure the income is mocked as being high.